mkfiles/stage1.mk

```diff
@@ -42,6 +43,7 @@ include $(ISO_SOURCE)/conf/modules.stage1
 
 $(ISO_TARGET)/.stage1: stage1-toolchain
 	@echo stage1-build
+	@grep LUNAR_ALIAS_OSSL $(ISO_TARGET)/etc/lunar/local/config || echo 'LUNAR_ALIAS_OSSL="openssl"' >> $(ISO_TARGET)/etc/lunar/local/config
```

Comment:
Why don't you add the alias to `template/etc/lunar/local/config` (and if needed `lunar-install/sbin/lunar-install`) instead?
This feels much cleaner to me